### PR TITLE
Text Utilities component

### PR DESCRIPTION
**Pull request by malaykhakhar**

### CODE REVIEW

File: components/Text Utilities/text_utilities.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        <p class="main-para">Below is a gray text</p>
+        <p class="text-gray">This text will appear in gray color</p>
+        <br><br>
+        <h1>Centre text</h1>
+        <p class="main-para">Below is a centered text</p>
+        <div class="text-centre">This text will appear in the centre</div>
+
+        <iframe
+          src="https://carbon.now.sh/embed/yp5S7ekMYo5ugHrQFfpt"
+          style="width: 680px; height: 540px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (Rishabh-078) [inline components/Text Utilities/text_utilities.html:69]: Inline styling being used, that needs to be avoided

**Review by Rishabh-078 — CHANGES_REQUESTED** [review]
Rest code looks fine to me.
From a UI point of view: examples of headings, smaller text, gray text, etc can be put inside a container so that they become distinct and easily noticeable.

File: components/Text Utilities/text_utilities.html (hunk 1 above)
Comment (malaykhakhar) [reply components/Text Utilities/text_utilities.html:69]: Hey so this is actually taken from a external website carbon.now.sh to display code in prettier way. So it was advisable not to change stuff from it, hence i have put it as it is. Rest i have not used inline styling anywhere

**Review by Rishabh-078 — APPROVED** [review]
Looks fine. You can merge it

**Review by manish-GH — APPROVED** [review]
The code looks good, it can be merged

**Pull request merged**